### ISSUE TITLE
QMs ruling "out of context" too quickly

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#44]
Ruling a quizzer "out of context" too early comes with many potential negatives. If the QM is incorrect, the question HAS to be redone, regardless of if the quizzer had 1 second left of their time or 29, and regardless of if the quizzer was "close" to getting the answer, or super far away. 

Waiting for the 30 seconds to expire often means the QM never has to make the judgment call of whether the quizzer went out of context...if the quizzer never got the correct answer.

**Comment by josiah-leinbach** [comment]
@scottpeterson Do you want to progress this proposal? While I understand the desire to protect quizzers from a hasty QM judgment, I also think that making a quizzer who has given an answer that is clearly out of context wait through the whole 30 seconds an unnecessary drain on them. Not to mention the time added to the quiz length from having to wait.

Also, if a QM is ruling the quizzer out of context before their 30 seconds have expired, presumably the QM and the AJ have verified this.

**Comment by scottpeterson** [comment]
Yep I still stand by this proposal for a best practices doc that QMs should be extremely sure of their ruling if they're going to stop the quizzer before 30 seconds have elapsed.

**Comment by jswingle** [comment]
This definitely does belong in best practices over rules, but I'd support it if we had a language proposal. 

I think my threshold for ending the 30sec period is much higher when QMing at higher levels of competition. At IBQ I will generally allow a quizzer to keep going for 30sec until they're either completely word perfect or undebatably incorrect/out of context. Such as, beginning to quote the wrong verse on a Quote question, violating the Trinity (formerly Deity) rule, etc. but otherwise I allow the full 30 seconds so as to stave off challenges. 

This is sensible to me, but maybe not to everyone, so should be in best practices.
